test/test_jit.py

```diff
@@ -9843,27 +9843,11 @@ def forward(self, x, y):
 }
 
 EXCLUDE_SCRIPT_MODULES = {
+    'test_nn_NLLLoss_weights_ignore_index_neg',
```

Comment:
Why was this added ?

Reply:
It looks like it's failing in aten for some case that wasn't being tested before